src/constants/skip.ts

```diff
@@ -0,0 +1,33 @@
+export type RouteStatus = {
+  chainId: string | undefined;
+  txHash: string | undefined;
+  status: string | undefined;
```

Comment:
do we know the actual string values this can be? Kind of seems like it below, might be cool to put it in the types.

Reply:
let me take a look - i *think* i did this for a reason but it's been a little while

Reply:
updated!